### PR TITLE
295299

### PR DESCRIPTION
**Pull request by KNyce71**
Minor edits in the BACnet Guide - part of the Adapter Doc Overhaul work. 

Thanks!

### CODE REVIEW

File: content/configuration/configuration.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The examples in the configuration topics use `curl`, a commonly available tool o
 
 For more information on PI Adapter configuration tools, see [Configuration tools](xref:ConfigurationTools).
 
-## Quick start
+## Quick Start
 
 This Quick Start guides you through setup of each configuration file available for PI Adapter for BACnet. As you complete each step, perform each required configuration to establish a data flow from a data source to one or more endpoints. Some configurations are optional.
```
[the remark this answers is not in the document]
Comment (osisoft-mbishop) [reply content/configuration/configuration.md:15]: Just trying to avoid confusion based on the heading.

**Review by osisoft-mbishop — CHANGES_REQUESTED** [review]
Hey  @KNyce71 , I have a few nitpicks, but your changes look good for the most part.

File: content/health/adapter-health-for-bacnet.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Dynamic data is sent every minute to configured health endpoints.
 The following health data are available:
 
 - [Device status](xref:DeviceStatusForBACnet)
-- [Next Health Message Expected](xref:NextHealthMessageExpectedForBACnet)
```
[the remark this answers is not in the document]
Comment (KNyce71) [reply content/health/adapter-health-for-bacnet.md:16]: Great advice. Much appreciated.

**Review by KNyce71 — COMMENTED** [review]
Hi @osisoft-mbishop....thanks for the quick review! I changed the capital S in "Quick start" to lowercase. I think I've gotten all of your other comments as well. If you could finalize, I would appreciate it.

**Review by osisoft-mbishop — APPROVED** [review]

**Pull request merged**